README.md

```diff
@@ -78,7 +84,26 @@ development: {
 
 #### Next, you will want to set up your database and start your server.
 
-Running `knex migrate:latest ` in the `server/` directory will use the migration files in `server/migrations` to create and format the tables so that they will work with wikonnect.
+Ensure that knex is installed globally to run some of the commands below
+Using npm:
+
+```
+npm install -g knex
+```
+
+Also ensure that you have redis-server running locally in your pc, To install it in your pc run the command below
```

Comment:
You could provide link for MacOS and Windows, may be just https://redis.io/download

Reply:
sure